claim/README.md

```diff
@@ -83,21 +97,38 @@ can trust the endpoint if necessary.
 Claiming can be performed through a SOCKS5 proxy. To do this when calling the script directly supply the proxy
 endpoint as:
 
-```
-netdata-claim.sh -token=MYTOKEN1234567 -rooms=room1,room2 -proxy=socks5h://127.0.0.1:11081
+```bash
+netdata-claim.sh -proxy=socks5h://127.0.0.1:11081 -token=MYTOKEN1234567 -rooms=room1,room2
 ```
 
-When claiming via the `netdata` binary set the following options in the config:
-```
+When claiming via the `netdata` binary, set the following options in your `netdata.conf` file:
+
+```conf
 [cloud]
     proxy = socks5://X.X.X.X:YYYY
 ```
+
 Proceed to claim using the command-line syntax:
-```
+
+```conf
 /usr/sbin/netdata -D -W "claim -token=MYTOKEN1234567 -rooms=room1,room2"
 ```
 
-Please note - if you supply the proxy endpoint in the configuration then it will also be used to tunnel
-the agent cloud link as well.
+> Please note: if you supply the proxy endpoint in the configuration then it will also be used to tunnel the agent cloud
+> link as well.
+
+## Unclaim (remove) an agent from Netdata Cloud
+
+The best method to remove an agent from Netdata Cloud is to unclaim it by deleting the `claim.d/` directory in your
+Netdata configuration directory.
+
+```bash
+cd /etc/netdata   # Replace with your Netdata configuration directory, if not /etc/netdata/
+rm -rf claim.d/
+```
+
+> You may need to use `sudo` or another method of elevating your privileges.
+
+Once you delete the `claim.d/` directory, the ACLK disconnects and Netdata Cloud removes the node from your account.
```

Comment:
I don't think this is true - is there any functionality in the cloud that does this?

Reply:
I think disconnection is not happening. It will just not connect again/next time.